Source/BluetoothManager.swift

```diff
@@ -70,11 +70,8 @@ public class BluetoothManager {
     /// Creates new `BluetoothManager` instance with specified implementation of `RxCentralManagerType` protocol which will be
     /// used by this class. Most of a time `RxCBCentralManager` should be chosen by the user.
     /// - parameter centralManager: Implementation of `RxCentralManagerType` protocol used by this class.
-    /// - parameter queueScheduler: Scheduler on which all serialised operations are executed (such as scans). By default main thread is used.
-    init(centralManager: RxCentralManagerType,
-         queueScheduler: SchedulerType = ConcurrentMainScheduler.instance) {
+    init(centralManager: RxCentralManagerType) {
```

Comment:
Update comment in line 103 of the current file regarding the init method.

Reply:
Done :)